init-share.md

```diff
@@ -42,7 +42,7 @@ Pull in Unison projects to be hosted on Unison Share
 .> pull.without-history https://github.com/unisonweb/share-pre-oct-2021:.contrib.thoradam .thoradam
 .> pull.without-history https://github.com/ChrisPenner/unison-codebase:.share.file .chrispenner.file
 .> pull.without-history https://github.com/iamevn/unison-code:.trunk .iamevn
-.> pull.without-history https://github.com/hagl/dhall-unison:.dhall.releases._v1_share .hagl.dhall
+.> pull.without-history https://github.com/hagl/dhall-unison:.dhall.releases._latest .hagl.dhall
```

Comment:
I'll make the  `.dhall.releases._latest` namespace be a copy of the current release whenever I release a new version. That way I'd get a possible update on Share whenever the Share DB is rebuilt. 
I hope that approach is ok?

Reply:
yeah thats great!

Reply:
I'd suggest adding a version number to your README if you don't have one already